ldfdata-service/src/main/resources/application.yaml

```diff
@@ -1,16 +1,18 @@
 spring:
   kafka:
-    stream:
-      input:
-        ldfdata:
-          topic-name: nbs_state_defined_field_data
-      output:
-        ldfdata:
-          topic-name-reporting: nrt_ldf_data
+    input:
+      topic-name: nbs_state_defined_field_data
+    output:
+      topic-name-reporting: nrt_ldf_data
+    dlq:
+      retry-suffix: _retry
```

Comment:
should this be `-retry` and `-dlt` as discussed

Reply:
DLQ and retry for `nrt_` topics is happening on the post-processing service, not in preprocessing, so we're good here